### ISSUE TITLE
Assets marked as Unsupported for no reason

### ISSUE THREAD

**Issue by gigo6000** [cloudinary/cloudinary_wordpress#416]
I haven't been able to sync my assets to Cloudinary I've tried the Auto sync method in the UI and also cli command but no luck, this is the output of the cli:

```
Analyzing 12 assets:
12 Analyzed |  100% [====================================================================================================================================================================================================================================] 0:00 / 0:00

Synced      : 2
Un-synced   : 0
Unsupported : 10
```

The 2 synced files are files I uploaded through the Wordpress admin after installing the plugin but the others are files that were already there when I installed the plugin. All the 10 files are valid jpgs so not sure they are marked as unsupported.

Wordpress Version: `5.7.1`
Cloudinary Plugin: `2.7.1`
PHP:  `7.4.1`
Ubuntu: `18.04.3 LTS`
Vagrant: `2.2.9`

**Comment by Vdeub-cloudinary** [comment]
Hi @gigo6000 we know we have some inconsistencies with the `WP-CLI`, if all the assets have a green icon, you should find them also in your Cloudinary Media Library and consider them as synced.
Please let me know if it is not the case.

**Comment by gigo6000** [comment]
Hi @Vdeub-cloudinary, I found the issue, the plugin is comparing the GUID domain with the Wordpress domain in this function: `is_local_media()` and some of the assets where I uploaded on another Wordpress domain ( we have different environments: `dev`, `local`, etc). I had to replace the domain in the GUID :

```sql
UPDATE `wp_posts` SET guid = REPLACE(guid, 'oldsiteurl', 'newsiteurl') WHERE guid LIKE 'oldsiteurl%'; 
```
After doing this and running: `wp cloudinary sync` I could get all the assets synced:

```
Synced      : 12
Un-synced   : 0
Unsupported : 0


All assets synced.
```

Is there any reason to do this check? I think a better check is to see if the file is accesible from the system, thanks.

**Comment by Vdeub-cloudinary** [comment]
Hi @gigo6000, based on your use case, this is expected.  Only local assets are supposed to be part of the sync system, so if you uploaded assets on another domain, it will be considered as `unsupported`. You can still use your workaround in your case if you want to have a consistent result for your instance

**Comment by gigo6000** [comment]
@Vdeub-cloudinary I think I disagree they should not be considered as `unsupported`, the assets are in the local system they were just uploaded on another domain at least users should have the option to bypass this check.

**Comment by robclancy** [comment]
I just wasted a lot of time working out what was going on with our sync. This really should be documented somewhere. And at the very least have text in the wp cli showing what unsupported means.

**Comment by Vdeub-cloudinary** [comment]
@robclancy Thanks for your feedback, we have a section related to syncing external assets in our documentation: https://cloudinary.com/documentation/wordpress_integration#syncing_external_assets - but we will take your comment into account to add more details on the `wp-cli` part.
Thanks!

**Comment by Vdeub-cloudinary** [comment]
Hi @robclancy, we have just released `3.0.3`, could you please upgrade the plugin? You can. now use the `--verbose` keyword to get more details.
